packages/client/tests/functional/tracing-no-sampling/_matrix.ts

```diff
@@ -0,0 +1,32 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'sqlite',
+    },
+    {
+      provider: 'postgresql',
+    },
+    {
+      provider: 'mysql',
+    },
+    {
+      provider: 'sqlserver',
+    },
+    {
+      provider: 'cockroachdb',
+    },
+    {
+      provider: 'mongodb',
+    },
+  ],
+  [
+    {
+      previewFeatures: '"tracing", "interactiveTransactions"',
+    },
+    {
+      previewFeatures: '"interactiveTransactions"',
+    },
+  ],
+])
```

Comment:
```suggestion
export default defineMatrix(() => [
  allProviders,
  [
    {
      previewFeatures: '"tracing", "interactiveTransactions"',
    },
    {
      previewFeatures: '"interactiveTransactions"',
    },
  ],
])
```

A handy constant of providers.

Reply:
Neat, thanks!